library/ix-dev/charts/radarr/questions.yaml

```diff
@@ -0,0 +1,225 @@
+groups:
+  - name: Radarr Configuration
+    description: Configure Radarr
+  - name: User and Group Configuration
+    description: Configure User and Group for Radarr
+  - name: Network Configuration
+    description: Configure Network for Radarr
+  - name: Storage Configuration
+    description: Configure Storage for Radarr
+  - name: Resources Configuration
+    description: Configure Resources for Radarr
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+  - variable: radarrConfig
+    label: ""
+    group: Radarr Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: instanceName
+          label: Instance Name
+          description: The name of the Radarr instance.
+          schema:
+            type: string
+            default: Radarr
+            required: true
+        - variable: additionalEnvs
+          label: Additional Environment Variables
+          description: Configure additional environment variables for Radarr.
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: env
+                label: Environment Variable
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: name
+                      label: Name
+                      schema:
+                        type: string
+                        required: true
+                    - variable: value
+                      label: Value
+                      schema:
+                        type: string
+                        required: true
+
+  - variable: radarrRunAs
+    label: ""
+    group: User and Group Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: user
+          label: User ID
+          description: The user id that Radarr will run as.
+          schema:
+            type: int
+            min: 1
+            default: 568
```

Comment:
Should it be immutable?

Reply:
It's the same thing as qbit.
The permissions container checks the parent dir here before chown